packages/replay/test/fixtures/performanceEntry/navigation.ts

```diff
@@ -33,13 +31,13 @@ export function PerformanceEntryNavigation(obj?: Partial<PerformanceNavigationTi
     domComplete: 682.5,
     loadEventStart: 682.5,
     loadEventEnd: 682.6999999284744,
-    type: 'navigate' as NavigationType,
+    type: 'navigate',
     redirectCount: 0,
     ...obj,
   };
 
   return {
     ...entry,
     toJSON: () => entry,
-  };
+  } as PerformanceNavigationTiming;
```

Comment:
(feel free to ignore)
l: Would this be an application for `satisfies` rather than the type cast? But not sure, was just asking myself why we need the cast here 😅

Reply:
actually, I can remove this cast overall by just making `entry` a const - that's better :D